src/crawlee/storages/_key_value_store.py

```diff
@@ -157,3 +157,15 @@ async def set_value(
             return await self._resource_client.delete_record(key)
 
         return await self._resource_client.set_record(key, value, content_type)
+
+    def get_public_url(self, key: str) -> str:
+        """Get the public URL for the given key.
+
+        Args:
+            key: Key of the record for which URL is required
+        Returns:
+            The public URL for the given key.
+        """
+        name = self.name or self._configuration.default_key_value_store_id
+        storage_dir = self._configuration.storage_dir
+        return f'file://{storage_dir}/storage/key_value_stores/{name}/{key}'
```

Comment:
I believe the `/storage` part is now redundant here.

Reply:
From test execution, I guess self._configuration.storage_dir doesn't contain /storage

Reply:
@janbuchar Yes the /storage part was redundant. I have removed it. File extension was missing and that has been added now